src/App.tsx

```diff
@@ -1,7 +1,43 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string | null
+};
+
+export class App extends React.Component<{}, State> {
+  state: State = {
+    pressedKey: null,
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.handleKeyUp);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.handleKeyUp);
+  }
+
+  handleKeyUp = (event: KeyboardEvent): void => {
+    this.setState({ pressedKey: event.key });
+  };
+
+  render() {
+    const { pressedKey } = this.state;
+
+    return (
+      <div className="App">
+        {pressedKey
+          ? (
+            <p className="App__message">
+              {`The last pressed key is [${pressedKey}]`}
+            </p>
+          )
+          : (
+            <p className="App__message">
+              Nothing was pressed yet
+            </p>
+          )}
```

Comment:
You don't need to write the same <p> tags twice - you should write one tag and add content depending on the condition.